applications/StructuralMechanicsApplication/custom_advanced_constitutive/constitutive_laws_integrators/high_cycle_fatigue_law_integrator.h

```diff
@@ -168,7 +167,8 @@ class HighCycleFatigueLawIntegrator
                                             const Properties& rMaterialParameters,
                                             double& rB0,
                                             double& rSth,
-                                            double& rAlphat)
+                                            double& rAlphat,
+                                            double& rN_f)
```

Comment:
maybe you could use e.g. a `struct` instead, then you don't need to change the function signature each time you need to pass a new parameter
(this is a general suggestion, might be applicable also elsewhere)

Reply:
@AlejandroCornejo also made this suggestion. I would like to do it in a new PR if it is ok to both of you.